deploy/template.yaml

```diff
@@ -2838,6 +2859,41 @@ Resources:
       ComparisonOperator: GreaterThanThreshold
       TreatMissingData: notBreaching
 
+  IOExceptionAlarm:
+    Type: AWS::CloudWatch::Alarm
+    Condition: IsNotDevelopment
+    Properties:
+      ActionsEnabled: true
+      AlarmActions:
+        - !ImportValue sns-topics-AlarmTopic
```

Comment:
Can we use `alarm-alerts-topic` like the canary alarms - otherwise think this will go to PagerDuty which is probably overkill